src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -67,21 +78,24 @@ target_link_libraries(MR_TESTS_CPP_AUXILIARY PUBLIC csirf cgadgetron)
 add_executable(MR_PROCESS_TESTDATA ${CMAKE_CURRENT_SOURCE_DIR}/mrtests_prep_testdata.cpp)
 add_dependencies(MR_PROCESS_TESTDATA GENERATE_SIMULATED_TESTDATA)
 target_link_libraries(MR_PROCESS_TESTDATA PUBLIC MR_TESTS_CPP_AUXILIARY csirf cgadgetron)
+set(MR_PROCESS_TESTDATA_EXE "${CMAKE_CURRENT_BINARY_DIR}/MR_PROCESS_TESTDATA")
```

Comment:
I'm not very proud of this line, but I wasn't able to get the property `LOCATION` of the target `MR_PROCESS_TESTDATA` (at least in this gigantic CMake project)